src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  function add(n) {
+    calculator.result += n;
+  }
```

Comment:
[CODE KNOWLEDGE] Instead of using the `function` keyword inside the object, you can take advantage of the method shorthand syntax.

BAD EXAMPLE:
 add: function(n) {
   this.result += n;
 },

GOOD EXAMPLE:
 add(n) {
   this.result += n;
 },